assisted-events-scrape/events_scrape/events_scrape.py

```diff
@@ -22,6 +25,8 @@
 
 RETRY_INTERVAL = 60 * 5
 MAX_EVENTS = 5000
+DEFAULT_ENV_MAX_ERRORS = "100"
```

Comment:
we have 1500 clusters in production and we don't know yet how many problematic clusters with exception we have so I think that 100 is a low value. maybe we should start with a higher value - investigate the exceptions and then decide.

Reply:
This feature it's a "safety net". Whenever there are "many" failures, we'll restart the whole process. It's similar to what we had before (while True), but we will reset any state that we have in the application by restarting it.

Due to the "shuffle", eventually we will process every cluster (as we are doing now).

This said, I don't mind higher or lower number, as I'll keep a close eye on this and handle all errors.